restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -283,6 +284,132 @@ public void putDocPath(
+              defaultValue = "false")
+          @QueryParam("profile")
+          Boolean profile,
+      @Context HttpServletRequest request,
+      @Suspended AsyncResponse asyncResponse) {
+    // just delegate to patchPath with empty path
+    List<PathSegment> path = Collections.emptyList();
+    patchDocPath(
+        authToken, namespace, collection, id, path, payload, profile, request, asyncResponse);
+  }
+
+  @PATCH
+  @ManagedAsync
+  @ApiOperation(
+      value = "Update data at a path in a document",
+      notes =
+          "Merges data at the path with requested data, assumes that the data at the path is already an object. Note that operation is not allowed if a JSON schema exist for a target collection.")
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "OK", response = WriteDocResponse.class),
+        @ApiResponse(code = 400, message = "Bad request", response = ApiError.class),
+        @ApiResponse(code = 401, message = "Unauthorized", response = ApiError.class),
+        @ApiResponse(code = 403, message = "Forbidden", response = ApiError.class),
+        @ApiResponse(code = 422, message = "Unprocessable entity", response = ApiError.class),
+        @ApiResponse(code = 500, message = "Internal Server Error", response = ApiError.class)
+      })
+  @Path("collections/{collection-id}/{document-id}/{document-path: .*}")
+  @Consumes({MediaType.APPLICATION_JSON, MediaType.APPLICATION_FORM_URLENCODED})
+  @Produces(MediaType.APPLICATION_JSON)
+  public void patchDocPath(
+      @ApiParam(
+              value =
+                  "The token returned from the authorization endpoint. Use this token in each request.",
+              required = true)
+          @HeaderParam("X-Cassandra-Token")
+          String authToken,
+      @ApiParam(value = "the namespace that the collection is in", required = true)
+          @PathParam("namespace-id")
+          String namespace,
+      @ApiParam(value = "the name of the collection", required = true) @PathParam("collection-id")
+          String collection,
+      @ApiParam(value = "the name of the document", required = true) @PathParam("document-id")
+          String id,
+      @ApiParam(value = "the path in the JSON that you want to retrieve", required = true)
+          @PathParam("document-path")
+          List<PathSegment> path,
+      @ApiParam(value = "The JSON document", required = true)
+          @NotNull(message = "payload not provided")
+          @NotBlank(message = "payload must not be empty")
+          String payload,
+      @ApiParam(
+              value = "Whether to include profiling information in the response (advanced)",
+              defaultValue = "false")
+          @QueryParam("profile")
+          Boolean profile,
+      @Context HttpServletRequest request,
+      @Suspended AsyncResponse asyncResponse) {
+    // TODO Eric should we actually fail the PATCH if the table is not there?
```

Comment:
Hmm...maybe - I think the reason why we eagerly create the table is because PUT does so as well, but we could also say that PATCH requires that there already is data to patch onto...that probably makes more sense

Reply:
So I decided that I don't want to change current behavior, so that I don't break anybody.. Feel more comfortable keeping the existing behavior..